src/storage/local-project-storage-client.ts

```diff
@@ -0,0 +1,118 @@
+// Implementation of ProjectStorageClient backed by IndexedDB,
```

Comment:
@gunsch @ObsidianSnoo, this adds another data source and removes some of the focus of play as a dumb fiddle. I think we should get together next week and chat about realistic goals for play to avoid remaking the organic mistakes of Studio. Ryan and I chatted earlier in the year about needing to do some refactors for internal vs external features, single file support, and other topics but haven't had much time to work on it.